samples/listenForOrderedMessages.js

```diff
@@ -1,124 +0,0 @@
-// Copyright 2019-2020 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-/**
- * This application demonstrates how to perform basic operations on
- * subscriptions with the Google Cloud Pub/Sub API.
- *
- * For more information, see the README.md under /pubsub and the documentation
- * at https://cloud.google.com/pubsub/docs.
- */
-
-'use strict';
-
-// sample-metadata:
-//   title: Listen For Ordered Messages
-//   description: Demonstrates how to order messages coming from a topic.
-//     Please see "Publish Ordered Message" for the other side of this.
-//   usage: node listenForOrderedMessages.js <subscription-name> [timeout-in-seconds]
-
-let subscribeCounterValue = 1;
-
-function getSubscribeCounterValue() {
-  return subscribeCounterValue;
-}
-
-function setSubscribeCounterValue(value) {
-  subscribeCounterValue = value;
-}
-
-const outstandingMessages = {};
-
-async function main(
-  subscriptionName = 'YOUR_SUBSCRIPTION_NAME',
-  timeout = 1000
-) {
-  timeout = Number(timeout);
-
-  // [START pubsub_listen_ordered_messages]
```

Comment:
Just to verify - if we search for this on devrel/snippets, we're not breaking anything?

Reply:
@anguillanneuf ?

Reply:
Yes temporarily. It's okay because I'm preparing to submit a CL to fix docs right away.